### PR TITLE
Update Proton VPN icon

### LINKED ISSUE #8650: Proton VPN is outdated
### Brand Name

Proton VPN

### Official Resources for Icon and Color

https://proton.me/media/kit

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
protonvpn

**Issue:** closes #8650 

**Similarweb rank:** 12,168. See https://www.similarweb.com/website/protonvpn.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from Official Icons in Brand website https://proton.me/media/kit
Color: `#87acf1` Taken from averaging logo colors by square root method.

### CODE REVIEW

**Comment by LitoMore** [comment]
Duplicate of #7795

**Comment by LitoMore** [comment]
We can still review and merge this PR. And update the title in another PR and mark the title change as a breaking change.

**Comment by LitoMore** [comment]
The gap is hard to see in a badge. Could you try with a 2px gap? Personally I think 2px is better.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -9361,8 +9361,8 @@
         },
         {
             "title": "ProtonVPN",
-            "hex": "56B366",
-            "source": "https://protonvpn.com/press"
+            "hex": "87ACF1",
```
[the remark this answers is not in the document]
Comment (PratyushJayachandran) [reply _data/simple-icons.json]: Although I applied the suggestion, can we consider using average colors, say by root mean square method to come up to a representative color? Just a suggestion. I used this tool https://10015.io/tools/image-average-color-finder. I presume there might be other tools for the same.

**Comment by PratyushJayachandran** [comment]
> The gap is hard to see in a badge. Could you try with a 2px gap? Personally I think 2px is better.

I changed it to 1px as 2px seemed a lot. Please review.
protonmail

**Review by LitoMore — APPROVED** [review]
image

Looks good.

**Pull request merged**